vmdb/db/migrate/20150319115040_create_replication_controllers.rb

```diff
@@ -0,0 +1,18 @@
+class CreateReplicationControllers < ActiveRecord::Migration
+  def up
+    create_table :container_replication_controllers do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.belongs_to :ems, :type => :bigint
+      t.string     :resource_version
```

Comment:
I see a few fields here like creation_timestamp and resource_version that aren't in the parsing directly...do they come from `parse_base_item`?